src/core/game.js

```diff
@@ -107,6 +107,7 @@ export function Game(game) {
 
   return {
     ...game,
+    name: game.name.replace(' ', '-'),
```

Comment:
I would think that this is the only change that is necessary.  Why do we also need to change socketio.js?